controllers/openstack_ansibleee_controller.go

```diff
@@ -268,21 +273,21 @@ func (r *OpenStackAnsibleEEReconciler) jobForOpenStackAnsibleEE(instance *redhat
 	}
 
 	if !hasIdentifier {
-		identifier := names.SimpleNameGenerator.GenerateName(fmt.Sprintf("%s-", instance.Name))
+		identifier := instance.Name
```

Comment:
Yepp we cannot have randomly generated data in the job definition as it will change the hash of the job at every reconciliation.